src/components/BatchSearchForm.vue

```diff
@@ -373,6 +400,7 @@ export default {
     },
     async onSubmit() {
       try {
+        if (!this.projects?.length) return
```

Comment:
Is this necessary? If we have the `allow-blank=false` on the multiselect, couldn't we use the form validation to disable the submit button?